index.html

```diff
@@ -0,0 +1,441 @@
+---
+layout: default
+title: Let's start building your software 🗲 deranged
+---
+<style type="text/css">
+    .hello-section {
+        min-height: 100vh;
+        height: 650px;
+        padding: 100px;
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        align-items: flex-start;
+        background-color: #2c3e50;
+        background-image: linear-gradient(40deg, #2c3e50 30%, #455d75 90%);
+        color: #eaf1f7;
+        position: relative;
+    }
+    .hello-section .logo {
+        color: inherit;
+        text-decoration: inherit;
+        font-weight: 200;
+        font-size: 1.3em;
+    }
+    .hello-section .big-greeting {
+        text-align: left;
+        font-size: 2.5em;
+    }
+    .hello-section .big-greeting h1 {
+        font-size: 2em;
+        margin: 0;
+    }
+    .hello-section .big-greeting h2 {
+        font-size: 1.3em;
+        margin: 0;
+    }
+    .hello-section .quick-intro {
+        text-align: left;
+        max-width: 550px;
+        font-size: 1.05em;
+        line-height: 150%;
+        position: relative;
+    }
+    .hello-section .quick-intro:before {
+        content: '🗲';
+        position: absolute;
+        left: -1.5em;
+        top: 0;
+    }
+    .hello-section .more-button {
+        background-color: transparent;
+        background-image: url('/img/expand_more_white.png');
+        position: absolute;
+        border: none;
+        cursor: pointer;
+        bottom: 100px;
+        right: 100px;
+        height: 48px;
+        width: 48px;
+        animation: upanddown 2.5s linear infinite alternate;
+    }
+    @keyframes upanddown {
+        from { transform: translateY(0); }
+        to { transform: translateY(-18px); }
+    }
+</style>
+<section class="hello-section">
+    <a href="/" class="logo">deranged</a>
+    <div class="big-greeting">
+        <h1>So nice to finally meet you!</h1>
+        <h2>Let's start building your software&hellip;</h2>
+    </div>
+    <p class="quick-intro">
+        We build scalable software that supports businesses and makes their employees and customers happy.
+        We always start from your current business situation, and help you forward in the most efficient way possible.
+        With flexiblity and engagement, we are your dev team.
```

Comment:
consider changing to 'we are your development team" to ensure that customers know what we talk about.

Reply:
Because it concludes the rest of the text, I think the meaning is clear?

I would agree if the sentence was there on its own (as it was before -- bad!)